acl2020_tools/awscognito/cognito.py

```diff
@@ -98,7 +98,7 @@ def create_user(client, profile, user):
         print(f"User {user.email} exists")
         return error.response
     except client.exceptions.ClientError as error:
-        print(f"Fail to create user {user.email}: {error.response}")
+        print(f"Fail to create user {user.email}")
```

Comment:
can you leave this as before? I added this error message because I fail to create some users last night and I want to see the reason. (It seems to be some string issue though.) But it would still be useful to look at the error message just in case it happens again.

Reply:
ok.

Reply:
@hao-fang I keep the `{error.response}` in the printout.  And I added `--debug` option to the scripts as well so that the scripts will show error details if any:  For example:

```bash
Fail to add user example+long.name.aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa@gmail.com to group attendees
{
    "Message": "1 validation error detected: Value at 'username' failed to satisfy constraint: Member must have length less than or equal to 128",
    "Code": "InvalidParameterException"
}
```